### PR TITLE
The check for trends, comparing new point to average

### PR DESCRIPTION
**Pull request by lauraser**
The following check is performed on all trends currently available in the QC. The medium quality applied for >3sigma deviations, bad quality for >6sigma deviations.

### CODE REVIEW

File: Modules/TPC/src/CheckOfTrendings.cxx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+#include <fairlogger/Logger.h>
+#include <TCanvas.h>
+#include <TGraph.h>
+#include <TList.h>
+#include <TPaveText.h>
+
+#include <vector>
+#include <numeric>
+
+using namespace std;
```
Comment (wiechula) [inline Modules/TPC/src/CheckOfTrendings.cxx:29]: Please don't use this and `std::` where needed. You do it in most places anyhow.
```suggestion
```

File: Modules/TPC/src/CheckOfTrendings.cxx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    return result;
+  }
+
+  double x_last = 0., y_last = 0.;
+  g->GetPoint(NBins - 1, x_last, y_last);
+  const double* yValues = g->GetY();
+  const vector<double> v(yValues, yValues + NBins - 1);
+
+  const double sum = std::accumulate(v.begin(), v.end(), 0.0);
+  const double mean = sum / v.size();
```
Comment (wiechula) [inline Modules/TPC/src/CheckOfTrendings.cxx:56]: is check for division by zero needed?

**Review by wiechula — CHANGES_REQUESTED** [review]

**Review by wiechula — APPROVED** [review]

**Comment by wiechula** [comment]
I think the macos failures are again unrelated

**Comment by wiechula** [comment]
@Barthelemy , can this be merged?

**Comment by Barthelemy** [comment]
yes. just trying to fix the mac build at the same time. I fixed a conflict here.

**Comment by wiechula** [comment]
@Barthelemy , thanks! I didn't spot this.

**Pull request merged**